.github/workflows/build_helm_svc_defns.yml

```diff
@@ -28,7 +55,7 @@ jobs:
         if: ${{ startsWith(env.SOURCE_BRANCH,'rel-') || env.SOURCE_BRANCH == 'main' }}
         shell: bash
         run: |
-          echo "NEXUS_REPOS_HELM=tessell-repos-helm-component" >> $GITHUB_ENV
+          echo "NEXUS_REPOS_HELM=${{vars.NEXUS_PUSH_REPOS_HELM}}" >> $GITHUB_ENV
```

Comment:
Why do we need to set this variable?

Reply:
To change from development to component for line 83